libs/components/phone-field/testing/src/phonefield/phone-field-fixture.ts

```diff
@@ -103,48 +106,45 @@ export class SkyPhoneFieldFixture {
    * @param searchText The name of the country to select.
    */
   public async selectCountry(searchText: string): Promise<void> {
-    await this.openCountrySelection();
+    await this.#openCountrySelection();
 
-    const countryFixture = await this.getCountryFixture();
+    const countryFixture = await this.#getCountryFixture();
     await countryFixture.searchAndSelectFirstResult(searchText);
 
-    return this.waitForCountrySelection();
+    return this.#waitForCountrySelection();
   }
 
   /**
    * Gets a boolean promise indicating if the phone field is disabled.
    */
   public async isDisabled(): Promise<boolean> {
-    const disabled = this.phoneFieldInput.getAttribute('disabled');
-    return this.coerceBooleanProperty(await disabled);
+    const disabled = this.#phoneFieldInput.getAttribute('disabled');
+    return this.#coerceBooleanProperty(await disabled);
   }
 
   /**
    * Gets a boolean promise indicating if the phone field is valid.
    */
   public async isValid(): Promise<boolean> {
-    if (!(await this.hasFormControl())) {
-      throw new Error(`Form control not found.`);
-    }
```

Comment:
this case was never being hit, and i have no reason to believe it ever should or will be, and strict mode confirms this.